lib/common/client/middleware/response/facility_parser.rb

```diff
@@ -18,129 +18,152 @@ def parse_body(env)
             path_test = env.url.path.match(/(FacilitySitePoint_\w{3})/) if path_test.nil?
             path_part = path_test[1]
             facility_map = facility_klass(path_part).attribute_map
-            env.body['features'].map { |location_data| build_facility_attributes(location_data, facility_map) }
+            env.body['features'].map do |location_data|
+              TempFacility.new(location_data, facility_map).build_facility_attributes
+            end
           end
 
-          def build_facility_attributes(location, mapping)
-            facility = { 'address' => {},
-                         'services' => {},
-                         'lat' => location['geometry']['y'],
-                         'long' => location['geometry']['x'] }
-            facility.merge!(make_direct_mappings(location, mapping))
-            facility.merge!(make_complex_mappings(location, mapping))
-            facility.merge!(make_address_mappings(location, mapping))
-            facility.merge!(map_benefits_services(location, mapping, facility['unique_id'])) if mapping['benefits']
-            facility.merge!(make_hours_mappings(location, mapping))
-            facility.merge!(map_health_services(location, mapping, facility['unique_id'])) if mapping['services']
-            facility
+          def facility_klass(path_part)
+            BaseFacility::PATHMAP[path_part]
```

Comment:
Does `facility_klass` get used anywhere else? Can we just get ride of this method and use `BaseFacility::PATHMAP[path_part]` directly inline?

Reply:
We can get rid of it, but I like the `facility_klass` method because it adds context to this code, and an explanation for what it's for. Do you feel strongly that I get rid of it?

Reply:
If I felt strongly, I'd deny the review ❌ . Otherwise, consider all feedback as suggestions that you can use or not use.

Reply:
Ok, thank you!